Makefile

```diff
@@ -274,6 +274,15 @@ install-local: docker-build docker-push-local
 	--set serviceAccount.create=false --set serviceAccount.name=${SERVICE_ACCOUNT} \
 	--set image.repository=${LOCAL_IMAGE} --set image.tag=latest --set image.pullPolicy=Always
 
+.PHONY: install-latest
```

Comment:
can we change the name to install-test or install-latest-test? as we are installing from the test ECR.